itou/job_applications/tests.py

```diff
@@ -44,6 +45,24 @@ def test_accepted_by(self):
         self.assertEqual(job_application.accepted_by, user)
 
 
+class JobApplicationQuerySetTest(TestCase):
+    def test_created_in_past_hours(self):
+
+        now = timezone.now()
+        hours_ago_10 = now - timezone.timedelta(hours=10)
```

Comment:
je chipote mais j'aurai mis `ten_hours_ago` pour bien se lire en anglais

TIL a variable name in python cannot start with a digit :-(